app/models/bot_user.rb

```diff
@@ -38,12 +38,11 @@ def confirmation_required?
 
   private
 
-  def set_uuid
-    self.uuid = ('check_bot_user_' + SecureRandom.hex(10))
-  end
+  # def set_uuid
+  #   self.uuid = ('check_bot_user_' + SecureRandom.hex(10))
+  # end
```

Comment:
Why this method is commented?

Reply:
No need to `uuid` as I removed it from `User` mode and use `uid` in `Account` and filled it for social account login only, will remove it from code

Reply:
Yeah, in that case please add a migration to remove this column from the database too